src/commands/test.js

```diff
@@ -53,16 +53,19 @@ function getPlatform() {
   const type = os.type();
   const arch = os.arch();
   const release = os.release();
+  const cpuCore = os.cpus()[0];
   const majorVersion = release.substr(0, release.indexOf('.'));
```

Comment:
After a second look, I just realize you can use `semver` for this, we have the package included into the project already 🙂  (https://github.com/graphprotocol/graph-cli/blob/master/package.json#L25).

I know it's not a super improvement, is more to be safe and use a tested lib that's already here.

Reply:
Not sure what you mean here. Isn't semver used for versioning? How would I use it to determine the cpu core model?

Reply:
Oh, I guess you mean only the last row from those 5. I have pushed that change as well.